### ISSUE TITLE
Richer color picker experience

### ISSUE THREAD

**Issue by fabiospampinato** [sindresorhus/System-Color-Picker#22]
I've just moved away from using [ColorSnapper2](https://colorsnapper.com/), and I think this app should implement some of the extra features that ColorSnapper2 provides for its color picker, as they improve the user experience significantly.

#### Features

1. Being able to resize the loupe with the scroll wheel. This is pretty useful as the current size of the loupe feels very small to me, it feels a bit disorienting, as if I can't see enough of the surrounding to orientate myself easily, I'd like to make it bigger.
2. Current color tooltip. I like this a lot because it allows the color picker to be used for more things, for example I can check if something is pure black or if two things are of the same color super quickly, currently doing the same with this app involves opening the window, which just breaks the workflow for me.
3. Close-up mode. When pressing `ctrl` on ColorSnapper2 the loupe sort of zooms in into the screen, showing a bigger loupe, much bigger pixels,  a pixels grid and importantly also the loupe moves more slowly. Those are pretty useful features IMO as picking a precise pixel currently with the app is not a smooth as it should be, it's just too easy to move the cursor accidentally to a neighbor pixel and I find that I physically get closer to the screen just to have a better view of the pixels.

#### Demo

https://user-images.githubusercontent.com/1812093/141295302-d8fcfd2c-ad11-4d8b-9d1d-ef9434d821f7.mov

**Comment by sindresorhus** [comment]
Yup, all of these are good suggestions and already planned. However, it requires making the color loupe from scratch, which is quite a lot of work.

Related issue: https://github.com/sindresorhus/System-Color-Picker/issues/16

**Comment by sindresorhus** [comment]
Tip: You can press Space to show the RGB color in the picker. It's not configurable though.

**Comment by fabiospampinato** [comment]
Nice! That improves the tool a lot for me.

**Comment by homatyano** [comment]
Another vote for the close-up mode. This feature is vital for those who use a graphic tablet like me.

The reason is that a click with a tablet is most of the time a short stroke, which means that with a tablet it's really difficult to "pick a pixel," so to say. So the ability to slow down the mouse cursor is indispensable.
